demos/calibration/calibration-pubmed-node-classification.ipynb

```diff
@@ -0,0 +1,3065 @@
+{
```

Comment:
<p>The w-* features of Pubmed are TF/IDF weighted word vectors. From this description it seems like these are binary vectors. I don't see any pre-processing step to make them binary either. </p>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Before cell 22